config/environments/development.rb

```diff
@@ -15,7 +15,7 @@
 
   # Don't care if the mailer can't send
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.delivery_method = :test
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = false
```

Comment:
It seemed to work for me when I set `perform_deliveries = true` in `development.rb` and if I kicked off delayed_job (`script/delayed_job run`). If it works for you, I'm cool with trying this out for a little while.